fem/tmop.cpp

```diff
@@ -1363,6 +1554,207 @@ void TMOP_Integrator::AssembleElementGrad(const FiniteElement &el,
    delete Tpr;
 }
 
+void TMOP_Integrator::SetFDPar(int fdflag_, int sz)
```

Comment:
Probably no need to require size here, as you can add them anyway in `Assemble...`. Simplifies the interface.

What is `Par`? Why not `SetFDFlag`?

Reply:
Par is for Parameters because I was initially allowing the order of approximation to be set as well. That is no longer the case though, so I will change it to SetFDFlag.